MapboxDirections/MBBannerInstructions.swift

```diff
@@ -0,0 +1,39 @@
+import Foundation
+
+@objc(MBBannerInstruction)
+public class BannerInstruction: NSObject, NSSecureCoding {
```

Comment:
MapboxDirections.swift shouldn’t assume a particular user interface paradigm. Are we sure this information will only ever be used for a turn banner, as opposed to a step table, local notification, or HUD? Let’s call this class VisualInstruction, unless there’s an overriding reason to refer to banners specifically.